cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -3,247 +3,378 @@
+    }
 
-        &_name {
-            .respond-to-min(@bp-sm-min, {
-                .h4();
-                margin-bottom: unit(15px / @font-size, em);
-                text-align: center;
-            });
-        }
+    // Expandable Overrides
+    // TODO: Remove these when base expandable is created.
+    .m-expandable_target {
+        padding: 0;
+        border: none;
     }
 
-    &_role {
-        padding: @grid_gutter-width / 2;
-        padding-right: @grid_gutter-width;
-        position: relative;
-        background: @gray-10;
-        border-color: @gray-20;
-        border-style: solid;
-        border-width: 1px 0 1px 0;
+    .m-expandable_content-animated {
+        padding: 0;
+    }
 
-        .respond-to-min(@bp-sm-min, {
-            border-width: 1px 0 0 0;
-        });
+    .m-expandable_cue-label {
+        display: inline-block;
     }
 
-    &_node {
-        width: 100%;
-        margin-bottom: -1px;
+    .m-expandable_header-left {
+        width: 73%;
+    }
 
-        .respond-to-min(@bp-sm-min, {
-            position: relative;
-            margin-bottom: @grid_gutter-width / 2;
-        });
+    .m-expandable_header-right {
+        width: 27%;
+    }
+}
+
+.org-chart_branch {
+    /* Borders on .org-chart_branch pseudo elements (:after on first branch, :before on last)
+       draw lines connecting the branch headers to the line from .org-chart_root.
+    */
+    .respond-to-min( @bp-sm-min, {
+        .grid_column( 6 );
+        position: relative;
+        padding-top: @margin__em;
+        border-width: 0;
+
+        &:before,
+        &:after {
+            // Whitespace in content so element can have dimensions set.
+            content: '';
+            position: absolute;
+            top: 0;
+            width: 50%;
+            height: @margin__em;
+            border: 0 solid @spine-color;
+        }
 
-        // nested .org-chart_node blocks have different background color & borders.
-        .org-chart_nodes {
-            border-top: 5px solid @pacific-60;
+        &:first-child {
+            padding-right: @grid_gutter-width / 2;
 
-            .org-chart_node {
-                .respond-to-min(@bp-sm-min, {
-                    margin-bottom: 0px;
-                });
+            &:after {
+                left: 50%;
+                border-width: @spine-width__px 0 0 @spine-width__px;
             }
+        }
 
-            .org-chart_role {
-                border-width: 1px 0 1px 0;
+        &.last-child {
+            padding-left: @grid_gutter-width / 2;
+
+            &:before{
+                right: 50%;
+                border-width: @spine-width__px @spine-width__px 0 0;
+            }
 
-                .respond-to-min(@bp-sm-min, {
-                    background: @pacific-20;
-                    border-color: #E5F1FA;
-                    border-width: 0 0 1px 0;
-                });
+            .org-chart_node:nth-last-child(2):after {
+                border:none
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/dca185a0d8c0205efa146b0695e88fa92ea3df97.